<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SlidePanel</class>
 <widget class="QWidget" name="SlidePanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>238</width>
    <height>659</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <property name="spacing">
    <number>180</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <property name="spacing">
      <number>15</number>
     </property>
     <item>
      <widget class="QGraphicsView" name="photo_view">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="10,1">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLineEdit" name="nickname_edit"/>
       </item>
       <item>
        <widget class="QPushButton" name="change_nickname_button">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="spacing">
      <number>25</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLineEdit" name="prev_passwd">
         <property name="placeholderText">
          <string>Previous password:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="new_passwd">
         <property name="placeholderText">
          <string>New password:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="confirm_passwd">
         <property name="placeholderText">
          <string>Confirm new password</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="change_passwd_button">
         <property name="text">
          <string>Change Password</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="change_passwd_label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>&lt;a href=&quot;my_click&quot;&gt; &lt;span style=&quot;color:blue; text-decoration:underline;&quot;&gt;Change Password&lt;/span&gt; &lt;/a&gt;</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="clear_cache_button">
         <property name="palette">
          <palette>
           <active>
            <colorrole role="ButtonText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>237</red>
               <green>51</green>
               <blue>59</blue>
              </color>
             </brush>
            </colorrole>
           </active>
           <inactive>
            <colorrole role="ButtonText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>237</red>
               <green>51</green>
               <blue>59</blue>
              </color>
             </brush>
            </colorrole>
           </inactive>
           <disabled/>
          </palette>
         </property>
         <property name="text">
          <string>Clear Cache</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
